docs/Maintainers/New-Season.md

```diff
@@ -1,3 +1,7 @@
+## Pre-Competition Season
+
+Within `src/backend/common/helpers/matchstats_helper.py`, add a new entry `2022: {}` (replace `2022` with the given upcoming year) to `COMPONENTS` for future custom component OPRs.
```

Comment:
Can we add a note about adding custom components as well? Just like - custom components can have human-readable names, can be calculated based on score breakdowns, etc. In 2020 it made sense to have this component OPR because XYZ